src/riak_kv_delete.erl

```diff
@@ -101,6 +100,19 @@ delete(ReqId,Bucket,Key,Options,Timeout,Client,ClientId,VClock) ->
             end
     end.
 
+new_maybe_ts_object(Bucket, {PK, LK}) ->
```

Comment:
Suggest naming this `create_tombstone_object` or similar and combining it with the next Tombstone binding line.